stubs/extensions/pdo.phpstub

```diff
@@ -0,0 +1,153 @@
+<?php
+
+class PDO
+{
+    public const PARAM_NULL = 0;
+    public const PARAM_INT = 1;
+    public const PARAM_STR = 2;
+    public const PARAM_LOB = 3;
+    public const PARAM_STMT = 4;
+    public const PARAM_BOOL = 5;
+    // public const PARAM_STR_NATL = 1073741824; since 7.2
```

Comment:
I'm not sure how to make this only available in 7.2, I'd rather not make a whole new stub file, but it doesn't look like psalm supports `@since`? Not immediately necessary anyway.

Reply:
Yeah, recurrent issue. It should not be an issue if we start generating stubs as we talked